```
conda.exceptions.ResolvePackageNotFound: 
  - doxygen
  - glibmm
During handling of the above exception, another exception occurred:
Traceback (most recent call last):
  File "/tmp/really-really-really-really-really-really-really-really-really-really-really-really-really-long-path/conda/bin/conda-build", line 11, in <module>
    sys.exit(main())
  File "/tmp/really-really-really-really-really-really-really-really-really-really-really-really-really-long-path/conda/lib/python3.7/site-packages/conda_build/cli/main_build.py", line 445, in main
    execute(sys.argv[1:])
  File "/tmp/really-really-really-really-really-really-really-really-really-really-really-really-really-long-path/conda/lib/python3.7/site-packages/conda_build/cli/main_build.py", line 436, in execute
    verify=args.verify, variants=args.variants)
  File "/tmp/really-really-really-really-really-really-really-really-really-really-really-really-really-long-path/conda/lib/python3.7/site-packages/conda_build/api.py", line 209, in build
    notest=notest, need_source_download=need_source_download, variants=variants)
  File "/tmp/really-really-really-really-really-really-really-really-really-really-really-really-really-long-path/conda/lib/python3.7/site-packages/conda_build/build.py", line 2343, in build_tree
    notest=notest,
  File "/tmp/really-really-really-really-really-really-really-really-really-really-really-really-really-long-path/conda/lib/python3.7/site-packages/conda_build/build.py", line 1334, in build
    output_metas = expand_outputs([(m, need_source_download, need_reparse_in_env)])
  File "/tmp/really-really-really-really-really-really-really-really-really-really-really-really-really-long-path/conda/lib/python3.7/site-packages/conda_build/render.py", line 746, in expand_outputs
    for (output_dict, m) in _m.copy().get_output_metadata_set(permit_unsatisfiable_variants=False):
  File "/tmp/really-really-really-really-really-really-really-really-really-really-really-really-really-long-path/conda/lib/python3.7/site-packages/conda_build/metadata.py", line 2047, in get_output_metadata_set
    bypass_env_check=bypass_env_check)
  File "/tmp/really-really-really-really-really-really-really-really-really-really-really-really-really-long-path/conda/lib/python3.7/site-packages/conda_build/metadata.py", line 721, in finalize_outputs_pass
    permit_unsatisfiable_variants=permit_unsatisfiable_variants)
  File "/tmp/really-really-really-really-really-really-really-really-really-really-really-really-really-long-path/conda/lib/python3.7/site-packages/conda_build/render.py", line 527, in finalize_metadata
    exclude_pattern)
  File "/tmp/really-really-really-really-really-really-really-really-really-really-really-really-really-long-path/conda/lib/python3.7/site-packages/conda_build/render.py", line 390, in add_upstream_pins
    permit_unsatisfiable_variants, exclude_pattern)
  File "/tmp/really-really-really-really-really-really-really-really-really-really-really-really-really-long-path/conda/lib/python3.7/site-packages/conda_build/render.py", line 378, in _read_upstream_pin_files
    permit_unsatisfiable_variants=permit_unsatisfiable_variants)
  File "/tmp/really-really-really-really-really-really-really-really-really-really-really-really-really-long-path/conda/lib/python3.7/site-packages/conda_build/render.py", line 154, in get_env_dependencies
    channel_urls=tuple(m.config.channel_urls))
  File "/tmp/really-really-really-really-really-really-really-really-really-really-really-really-really-long-path/conda/lib/python3.7/site-packages/conda_build/environ.py", line 758, in get_install_actions
    raise DependencyNeedsBuildingError(exc, subdir=subdir)
conda_build.exceptions.DependencyNeedsBuildingError: Unsatisfiable dependencies for platform linux-64: {'doxygen', 'glibmm'}
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -41,6 +41,8 @@ env:
   # SystemVerilog tools
   - PACKAGE=zachjs-sv2v         CONDA_CHANNELS="conda-forge"
   - PACKAGE=odin_II             CONDA_CHANNELS="pkgw-forge,conda-forge"
+  # protocol analyzers
+  - PACKAGE=sigrok-cli          CONDA_CHANNELS="conda-forge"
 
 script:
   - bash $TRAVIS_BUILD_DIR/.travis/script.sh

--- a/sigrok-cli/build.sh
+++ b/sigrok-cli/build.sh
@@ -1,0 +1,21 @@
+#! /bin/bash
+
+set -e
+set -x
+
+if [ x"$TRAVIS" = xtrue ]; then
+	CPU_COUNT=2
+fi
+
+for f in libserialport libsigrok libsigrokdecode sigrok-cli
+do
+	cd $f
+	./autogen.sh
+	./configure --prefix=$PREFIX
+	make -j$CPU_COUNT
+	make install
+	cd -
+done
+
+$PREFIX/bin/sigrok-cli -V
+find $PREFIX/share -name "__pycache__" -exec rm -rv {} +

--- a/sigrok-cli/meta.yaml
+++ b/sigrok-cli/meta.yaml
@@ -1,0 +1,46 @@
+package:
+  name: sigrok-cli
+  version: {{ GIT_FULL_HASH }}
+
+source:
+  - git_url: https://github.com/sigrokproject/libserialport
+    git_rev: master
+    folder: libserialport
+  - git_url: https://github.com/sigrokproject/libsigrok
+    git_rev: master
+    folder: libsigrok
+  - git_url: https://github.com/sigrokproject/libsigrokdecode
+    git_rev: master
+    folder: libsigrokdecode
+  - git_url: https://github.com/sigrokproject/sigrok-cli
+    git_rev: master
+    folder: sigrok-cli
+
+build:
+  number: {{ '1%04i00%s'|format(GIT_DESCRIBE_NUMBER|int, os.environ.get('DATESTR')) }}
+  string: {{ '%04i'|format(GIT_DESCRIBE_NUMBER|int) }}.{{ os.environ.get('DATESTR') }}.{{ GIT_DESCRIBE_HASH }}
+  script_env:
+    - CI
+    - TRAVIS
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - python {{ python }}
+    - pkg-config
+    - libusb
+    - libzip
+    - glib
+    - glibmm
+    - doxygen
+  host:
+    - python {{ python }}
+    - pkg-config
+    - libzip
+    - glib
+
+about:
+  home: http://sigrok.org/
+  license: GPLv3
+  summary: 'The sigrok project aims at creating a portable, cross-platform, Free/Libre/Open-Source signal analysis software suite that supports various device types (e.g. logic analyzers, oscilloscopes, and many more).'

